NAMESPACE

```diff
@@ -1 +1,5 @@
-# Generated by roxygen2: fake comment so roxygen2 overwrites silently.
+# Generated by roxygen2: do not edit by hand
+
+export(start_addin)
```

Comment:
Do these need to be exported? Mostly curious here

Reply:
I don't think so - if not then using the addin just calls `orderly.rstudio:::start_addin`